### PR TITLE
Package page jump

### LINKED ISSUE #501: Package page jump
> Hmm, can't reproduce it - checked in FF and Chrome - both act ok

Happens only sometimes it seems, and also on master so moving to a separate issue: https://dl.dropboxusercontent.com/s/i8uh47d86jfifz5/2023-02-11_15-28-08.gif

_Originally posted by @MartinKolarik in https://github.com/jsdelivr/www.jsdelivr.com/issues/498#issuecomment-1426785020_

### PR DESCRIPTION
**Pull request by xbpcb**
Fixes #501

### CODE REVIEW

File: src/views/components/package-header.html [hunk 1]
```diff
@@ -4,7 +4,7 @@
 			<div class="col-sm-8">
 				<div class="package-name-wrapper">
 					{{#if packageNameAsLink}}
-						<a href="/package/npm/{{name}}" on-click="@this.set('searchOpened', false)" class="package-name">{{name}}</a>
```
Comment (MartinKolarik) [inline src/views/components/package-header.html:7]: Is this related? 🤔

Comment (xbpcb) [reply src/views/components/package-header.html:7]: Nope, just redundant
